packages/ilios-common/app/styles/ilios-common/components/fade-text.scss

```diff
@@ -0,0 +1,48 @@
+@use "../colors" as c;
+
+.display-text-wrapper {
+  &.is-faded {
+    max-height: 200px;
+    overflow: hidden;
+    position: relative;
+  }
+}
+
+.fade-text-control {
+  bottom: 140px;
+  height: 130px;
+  padding-top: 140px;
+  position: relative;
+}
+
+span {
```

Comment:
The components in here aren't automatically wrapped in any way so this `span` is targeting every span in the application. Take a look at some of the other component sass and see how they use a top level class to contain all the styles and ensure they don't leak out into the rest of the app. 

Everything here should go under `.fade-text {`.